### DISCUSSION TITLE
Request to add Baikalos

### DISCUSSION THREAD

**Discussion by ahmedhridoy371** [ROM-builders/temporary#24119] [Q&A]
BaikalOs Are based on crdroid source with some unique features.
Source are fully stable and buildable also they provide regular updates.

here source link : https://github.com/baikalos/android

**Comment by ahmedhridoy371** [comment]
@Apon77 bhai kindly free time aa aita aktu check koiren

**Reply by hipexscape** [reply]
Not very popular rom

**Comment by Apon77** [comment]
non popular roms are not added.
